### ISSUE TITLE
Keyboard navigation between tabs is broken

### ISSUE THREAD

**Issue by ealmloff** [DioxusLabs/components#35]
The tabs component has some code for keyboard navigation with the arrow keys, and start/end keys, but it currently doesn't work in the demo when focusing one of the tabs
